lib/connect_proto/src/administered_substance.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+package primary.connect;
+option go_package = "primary.connect";
+
+import "google/protobuf/timestamp.proto";
+
+import "coded_value.proto";
+import "identifier.proto";
+
+message AdministeredSubstance {
```

Comment:
I think this was fine named `Substance`. It's just the field in `Order` that should be `administered_substances`
```protobuf
message Order {
  // ...
  message Order {
    // ...
    repeated Substance administered_substances = X;
  }
  // ...
}
```

Reply:
Ahhh okay